src/templates/ssr/interscroller/index.svelte

```diff
@@ -6,14 +6,17 @@
 	<img
 		src="[%TrackingPixel%]"
 		class="creative__pixel creative__pixel--displayNone"
+		alt="pixel"
```

Comment:
I don't think we should have an `alt` tag for these `img`'s. Tracking pixel's aren't displayed to the user so we should probably hide them from screen readers too.

Reply:
good point!

Reply:
I like the `aria-hidden` addition 👍